pkg/action/install_test.go

```diff
@@ -94,41 +95,30 @@ var _ = Describe("Install action tests", func() {
 	Describe("Install Action", Label("install"), func() {
 		var device, cmdFail string
 		var err error
-		var cmdline func() ([]byte, error)
 		var spec *v1.InstallSpec
 		var installer *action.InstallAction
 
 		BeforeEach(func() {
-			device = "/some/device"
+			os.RemoveAll("/tmp/test.img")
+			Expect(os.RemoveAll("/tmp/test.img")).ToNot(HaveOccurred())
+			// at least 2Gb in size as state is set to 1G
+			_, err = diskfs.Create("/tmp/test.img", 2*1024*1024*1024, diskfs.Raw, 512)
+			Expect(err).ToNot(HaveOccurred())
+			device = "/tmp/test.img"
```

Comment:
Move this definition above the `RemoveAll` and use it in `RemoveAll` too?

Reply:
yep, good point